src/jvm_wrapper/bridge/packed_color_array_bridge.h

```diff
@@ -6,6 +6,13 @@
 namespace bridges {
     PACKED_ARRAY_BRIDGE(PackedColorArrayBridge, Color, "godot.core.PackedColorArray$Bridge") {
         PACKED_ARRAY_BRIDGE_CLASS(PackedColorArrayBridge, Color)
+
+          // clang-format off
```

Comment:
Why format off and on here?
would make sense to format it here to be multiline no?

Reply:
because clang-format is a terrible tool